### ISSUE TITLE
error CS0246: The type or namespace name `MissionID' could not be found.

### ISSUE THREAD

**Issue by casper** [HearthstoneBot/HearthstoneBot#42]
Trying to build this with latest Assembly-CSharp.dll copied from the HS folder I get this error:

projects\Bot\src\AIBot.cs(91,17): error CS0246: The type or namespace name `MissionID' could not be found. Are you missing an assembly reference?

How do I fix this error?

**Comment by morefreeze** [comment]
@casper do you found 'MissionID'? I find the same issue.

**Comment by morefreeze** [comment]
@casper oh, I see. You should add reference of "Assembly-CSharp.dll" which locate in your Heartstone\Hearthstone_Data\Managed

**Comment by morefreeze** [comment]
But there are so many error here, I think the reason is the dll file has been updated.

**Comment by casper** [comment]
@morefreeze Yeah that's the problem. MissionID doesn't exist any more, and many other symbols are missing.
